drop.go

```diff
@@ -0,0 +1,61 @@
+package lo
+
+func Drop[T any](collection []T, n int) []T {
+	if len(collection) <= n {
+		return make([]T, 0)
+	}
+
+	result := make([]T, len(collection)-n)
+	for i := n; i < len(collection); i++ {
+		result[i-n] = collection[i]
+	}
+
+	return result
+}
+
+func DropWhile[T any](collection []T, predicate func(T) bool) []T {
+	result := []T{}
+
+	i := 0
+	for ; i < len(collection); i++ {
+		if !predicate(collection[i]) {
+			break
+		}
+	}
+
```

Comment:
Since you know the size of result, you should allocate it here, and use `result[j - i] = xxx` instead of `append`.

(prevent a little overhead)

Reply:
Done